### DISCUSSION TITLE
支付宝付款完成，未增加时长。

### DISCUSSION THREAD

**Discussion by lingzai777** [getlantern/lantern-forum-cn#911] [General]
支付宝付款完成，未增加时长。麻烦帮忙解决一下，谢谢。账号：460268141@qq.com
![IMG_2191](https://github.com/user-attachments/assets/f3fa18e7-0655-4f90-a192-6b847a8929be)

**Comment by lingzai777** [comment]
@Mike777Khoo 请帮忙核实解决下，谢谢

**Comment by MaxMKMax** [comment]
@lingzai777 好的. 我帮您提交问题上去. 谢谢您支持蓝灯!

**Comment by lingzai777** [comment]
@Mike777Khoo 您好，时长还未增加。距离付款已经过去20天了...

**Comment by lingzai777** [comment]
您好，时长还未增加。距离付款已经过去30天了...

**Reply by MaxMKMax** [reply]
@lingzai777 您好. 我再帮您提醒一下负责人. 不好意思让您久等了.

**Reply by greySkiesMarsh** [reply]
Hello @lingzai777 

I'm very sorry for the delay. Please redeem the following code in the app to get your account activated. 

KK6M4-V9J93-D7628-GHT9X-GTFFT

**Comment by lingzai777** [comment]
thanks，Problem Solved
&nbsp;


王霖
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月24日(星期四) 上午6:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 支付宝付款完成，未增加时长。 (Discussion #911)














 
Hello @lingzai777
 
I'm very sorry for the delay. Please redeem the following code in the app to get your account activated.
 
KK6M4-V9J93-D7628-GHT9X-GTFFT
